packages/paste-theme-designer/components/preview/FormPreview.tsx

```diff
@@ -36,11 +35,6 @@ const FormPreview: React.FC = () => {
           />
           <HelpText id={inputId('input-help-text')}>This is help text.</HelpText>
         </Box>
-        <Box>
-          <Label htmlFor={inputId('textarea')}>Textarea input</Label>
-          <TextArea aria-describedby={inputId('textarea-help-text')} id={inputId('textarea')} onChange={() => {}} />
-          <HelpText id={inputId('textarea-help-text')}>This is help text.</HelpText>
-        </Box>
```

Comment:
Same here. Was the identifier a fix for this too? Or is TextArea just a bit weird?

We've struggled in the past with textarea compilation FWIW so it might be an actual problem with textarea we need to investigate

Reply:
Yeah it may be related to the library the textarea depends on.  Adding this back broke the builds again.

Reply:
I wonder if that will go away with this https://github.com/twilio-labs/paste/pull/2454

I've seen it struggle with that library in the past with builds / compilation

Reply:
I've tried removing it from Textarea but the error persists. I'm completely at a loss.

Reply:
https://issues.corp.twilio.com/browse/DSYS-3418